### DISCUSSION TITLE
How to submit for help on netbox_lifecycle plugin

### DISCUSSION THREAD

**Discussion by chirpp2** [netbox-community/netbox#18167] [Help Wanted!]
I am trying to get this plugin to work and not geting far. How do i contact someone for support on this? it is not supported by Netbox so figure it is the author of the plugin. Am i missing something that i cannot see contact information for that?

**Comment by DanSheps** [comment]
Is it my plugin?  Then you need to go to my github for that.

**Reply by chirpp2** [reply]
Hello and thanks for the reply. yes it is your program that I am referring to. I am probably missing something as the install directions just say that I need to add it to the Plugin configuration area. There is no pip install or anything right?

**Reply by candlerb** [reply]
https://netboxlabs.com/docs/netbox/en/stable/plugins/installation/

**Reply by chirpp2** [reply]
Thanks for that.  I found a few more docs on line that showed some other steps, and apparently i was in the wrong virtual env when i was installing as well. 

Works good now.

**Reply by chirpp2** [reply]
DanSheps, I am entering the info in the Lifecycle area, and choosing from the drop down of the list of Device_types.  Cool! Once i add this data for End Of Sale, end of main, EoSupport, is there any chance to see that data tied to end devices that are using this device_type?  So if i go to an end deviec with this device_type, it can show there too? I see i can get this with API so could do it that way, i just wanted to know if that is an option or a silly idea?

**Reply by DanSheps** [reply]
You want to see the EoS information for the specific device type on the device?  Could you raise it as a FR in my GH repository?
